upgrade/upgrade.go

```diff
@@ -58,6 +58,17 @@ func Upgrade(kubeconfigPath, currentNodeID string) error {
 		return errors.Wrap(err, "unable to create scheme")
 	}
 
+	if err := upgradeAPIVersion(currentNodeID, namespace, config, lhClient, kubeClient); err != nil {
```

Comment:
Upgrade.go is designed to do upgrade between API group version. I am not sure this is the right place for this specific change. How about implementing this upgrade using fixup similar logics in data store instead?

Reply:
1.Upgrade.go can be considered as a general upgrade function for the existing Longhorn. And the API group upgrade part can be removed separately later.
2. I prefer not to modify the data store part for this kind of one-time upgrade. It's a little bit redundant and we may forget to remove it later. That's why I put it in upgrade.go rather than the controller or the data store part.

Reply:
OK, we can do this. But we shouldn't need a leader election for every upgrade case. Do it once and run the upgrade process there should be enough.